RRemoteConfig/ConfigModel.swift

```diff
@@ -29,11 +34,30 @@ internal struct KeyModel: Decodable, Parsable {
     let createdAt: String
 
     init?(data: Data) {
-        guard let dictionary = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String] else {
+        guard let dictionary = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String],
+            let id = dictionary["id"],
+            let key = dictionary["key"],
+            let createdAt = dictionary["createdAt"] else {
             return nil
         }
-        self.id = dictionary["id"] ?? ""
-        self.key = dictionary["key"] ?? ""
-        self.createdAt = dictionary["createdAt"] ?? ""
+        self.id = id
+        self.key = key
+        self.createdAt = createdAt
+    }
+}
+
+extension Data {
+    mutating func trailingNewlineTrimmed() -> Data {
```

Comment:
The name of this function sounds confusing to me, but maybe it's an iOS/Swift standard? `trimTrailiningNewline` is more clear to me. What do you think?

Reply:
Because it's an operation on self that returns self I thought `trimmed` was more natural naming.